optaplanner-examples/src/main/java/org/optaplanner/examples/common/app/CommonApp.java

```diff
@@ -151,4 +151,14 @@ protected AbstractSolutionExporter createSolutionExporter() {
         return null;
     }
 
+    public Solution_ loadUnsolvedProblemFromResource(String problemFileName) {
+        String pathToResource = String.format("/org/optaplanner/examples/%s/data/%s", dataDirName, problemFileName);
+        String fileName = CommonApp.class.getResource(pathToResource).getFile();
+        return createSolutionFileIO().read(new File(fileName));
+    }
+
+    public Solution_ loadUnsolvedProblemFromDataFolder(String problemFileName) {
```

Comment:
The method is unused. Let's remove it.

Reply:
I wrote it in advance for tests, that will be migrated in future. Basically, next one (TerminationTest) will use that method in every test case.

Reply:
OK.